core/src/test/resources/ssl-testing/server_key.pk8

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
well, our tests use them now (they had been checked in previously, I'm updating them because tests fail because the previous ones were expired. I suppose we could generate them as part of the test?